utils/example_hierarchical_abstraction.jl

```diff
@@ -155,7 +155,7 @@ AB.HierarchicalAbstraction.set_optimizer!(
 )
 
 using Suppressor
-@suppress begin # this is a workaround to supress the undesired output of SDPA
+@suppress begin # this is a workaround to supress the undesired output of Clarabel
```

Comment:
Maybe we don't need the supress anymore (I think I added it because SDPA was always printing some stuff)

Reply:
Thanks @egidioln I'll remove it